wire-ios-data-model/Source/Model/Message/Message.swift

```diff
@@ -88,6 +88,10 @@ public extension ZMConversationMessage {
         guard isSystem else { return false }
         return systemMessageData!.systemMessageType == .messageDeletedForEveryone
     }
+
+    var supportsCollapsing: Bool {
+        isFile || isAudio || isVideo || isLocation || isImage || isText
+    }
```

Comment:
suggestion: collapsing is only about presentation, we could move this property somewhere else (into the UI layer).
suggestion: rename
```suggestion
    var isCollapsingSupported: Bool {
        isFile || isAudio || isVideo || isLocation || isImage || isText
    }
```